src/ipmi.c

```diff
@@ -388,8 +388,12 @@ static int sensor_list_add(c_ipmi_instance_t *st, ipmi_sensor_t *sensor) {
         ipmi_sensor_get_event_support(sensor));
 
   /* Both `ignorelist' and `sensor_name_ptr' may be NULL. */
-  if (ignorelist_match(st->ignorelist, sensor_name_ptr) != 0)
-    return 0;
+  if (ignorelist_match(st->ignorelist, sensor_name_ptr) != 0) {
```

Comment:
This logic you have moved into L785, so this code is duplicate and will never be called.

Reply:
done